docs-ref-mapping/reference-unified.yml

```diff
@@ -1424,5 +1429,29 @@
     - name: Other
       uid: azure.nodejs.sdk.landingpage.services.other
       landingPageType: Service
-      children:
-      - '**'
+      items:
+      - name: Abort Controller
```

Comment:
My only concern, in `reference-unified` we have these `core` packages right. But if I look down below in `reference-latest` or `reference-unified`, I only see maybe two of these?